app/src/main/java/com/example/dynamicdiet/dto/Person.kt

```diff
@@ -1,5 +1,7 @@
 //person class with a constructor
 
-class Person(var name: String, var currentWeight: Int, var past: List<Int>, var diet: String){
-
+data class Person(var name: String, var currentWeight: Int, var past: List<Int>, var diet: String){
+    override fun toString(): String {
+        return ("$name $currentWeight $diet")
+    }
```

Comment:
change class to data class and override toString().
https://kotlinlang.org/docs/data-classes.html

Reply:
Good.